src/ruuvitag_ble/df3_decoder.py

```diff
@@ -24,13 +24,12 @@ def humidity_percentage(self) -> float | None:
 
     @property
     def temperature_celsius(self) -> float | None:
+        if self.data[2] == -128:
+            return None
         return round(self.data[2] + self.data[3] / 100.0, 2)
 
     @property
     def pressure_hpa(self) -> float | None:
-        if self.data[3] == 0xFFFF:
-            return None
-
```

Comment:
Hm?

Reply:
If we stick to the [protocol definition](https://github.com/ruuvi/ruuvi-sensor-protocols/blob/master/dataformat_03.md) by the letter, `65535` is a valid value for V3 but not for V5.  Kind of a technicality but meh.

Reply:
Gotcha, thanks for the clarification.